ttax/utils.py

```diff
@@ -0,0 +1,16 @@
+import jax
+import jax.numpy as jnp
+
+from ttax.base_class import TTBase
+from ttax.base_class import TT
+from ttax.base_class import TTMatrix
+from ttax.compile import WrappedTT
+
+def is_tt_tensor(arg) -> bool:
+  return isinstance(arg, TT) or isinstance(arg, WrappedTT)
```

Comment:
What if wrapperTT is wrapped around a TT-matrix? I think it should be `(isinstance(arg, WrappedTT) and not arg.is_tt_matrix)`

Reply:
Do you mean `arg.tt.is_tt_matrix`?